README.md

```diff
@@ -1,39 +1,46 @@
-   - Implement the database function calls in automation that [don't](https://github.com/mozilla-services/third-party-library-update/blob/master/automation.py#L27) [exist](https://github.com/mozilla-services/third-party-library-update/blob/master/automation.py#L38)
-     - For Hardcoded Database, they're just going to do nothing...
-   - Figure out how we're going to figure out who to needinfo/flag for review for the libraries
-     - Can this be determined from moz.build files? Or do we need to hardcode it in the database?
-   - Handle if [./mach vendor](https://github.com/mozilla-services/third-party-library-update/blob/master/components/mach_vendor.py#L6) fails, and then handle it
-     - We'll probably need to catch the Exception that's raised by run_command and then re-throw a more intelligent exception indicating the merge conflicts, ideally with detailed output.
-     - Before we re-throw the exception (so still inside mach_vendor.py.vendor), we should file a bug about the conflicts and then save the job to the database indicating this commit failed with a note about merge conflicts
- - Complicated
-   - Attach the phabricator patch. ([There's stuff that does this already in-tree, so there's something to vaguely copy or look at for reference.](https://searchfox.org/mozilla-central/source/taskcluster/docker/python-dependency-update/runme.sh))
-   - Figure out how real python applications are made these days and make this look like them. Something about requirements.txt?
-   - somewhere around [db.has_job](https://github.com/mozilla-services/third-party-library-update/blob/master/automation.py#L27) we should see if we had a job that failed due to merge conflicts, and if the bug we filed for those merge conflicts has been closed. If it hasn't, we should assume that future jobs would also fail, and skip this library.
- - Very Complicated
-   - Create process_existing_job
-     - Going to need to read the failed jobs off taskcluster
-     - Handle if the build job failed
-     - Handle if tests failed
-     - File comments on the bug and place a needinfo
-     - If stuff succeeded flag the patch for review
\ No newline at end of file
+<h1 align="center">
+  Updatebot
+</h1>
+
+[![<mozilla-services>](https://img.shields.io/circleci/build/gh/mozilla-services/updatebot?label=tests&style=flat-square)](https://circleci.com/gh/mozilla-services/updatebot)
+[![codecov](https://img.shields.io/codecov/c/gh/mozilla-services/updatebot?style=flat-square)](https://codecov.io/gh/mozilla-services/updatebot)
+
+Automated detection, patching, testing, and bug filing for updates in third-party libraries for Firefox.
+
+## 🐍 Initial Setup
+
+This project requires [Poetry](https://python-poetry.org/docs/) and a version of [Python](https://www.python.org/downloads/release/python-359/) at least greater than 3.5.
+
+You will also need to install [MySQL](https://www.mysql.com/downloads/).
```

Comment:
Perhaps "We talk to a database; currently MySQL is supported. Copy the local config file with `cp ...` and configure the database connection parameters. Updatebot will automatically create and populate the database with its structure and required data."